umbral-pre-python/umbral_pre/__init__.pyi

```diff
@@ -201,35 +208,34 @@ class VerifiedCapsuleFrag:
     def to_bytes_simple(self) -> bytes:
         ...
 
-
```

Comment:
Intentional?

Reply:
Moved to line 220. Sorry about these changes (moving stuff around), I think I accidentally formatted this file.